fs/fs.go

```diff
@@ -176,8 +176,8 @@ func processMounts(mounts []*mount.Info, excludedMountpointPrefixes []string) ma
 		if !strings.HasPrefix(mount.Fstype, "ext") && !supportedFsType[mount.Fstype] {
 			continue
 		}
-		// Avoid bind mounts.
-		if _, ok := partitions[mount.Source]; ok {
+		// Avoid bind mounts, exclude tmpfs.
+		if _, ok := partitions[mount.Source]; ok && mount.Fstype != "tmpfs" {
```

Comment:
can we please move `mount.Fstype != "tmpfs"` into a separate `if` condition?

Reply:
> can we please move `mount.Fstype != "tmpfs"` into a separate `if` condition?

Done